vscode/src/views/TestViewController.ts

```diff
@@ -312,7 +313,7 @@ export class NbTestAdapter {
             }
             const result = regExp.exec(line);
             if (result) {
-                message.appendText(result[1]).appendText('(').appendMarkdown(`[${result[3]}](command:${COMMAND_PREFIX}.open.stacktrace?${encodeURIComponent(JSON.stringify([currentTestUri, result[2], result[4], +result[5]]))})`).appendText(')');
+                message.appendText(result[1]).appendText('(').appendMarkdown(`[${result[3]}](command:${extConstants.COMMAND_PREFIX}.open.stacktrace?${encodeURIComponent(JSON.stringify([currentTestUri, result[2], result[4], +result[5]]))})`).appendText(')');
```

Comment:
Should we replace the command names with the references from *commands.ts*?

Reply:
Done